src/shared-api-objects/current-patient.ts

```diff
@@ -17,7 +17,7 @@ window.addEventListener("single-spa:routing-event", () => {
 });
 
 function getPatientUuidFromUrl() {
-  const match = /\/patient\/(.+)\/?/.exec(location.pathname);
+  const match = /\/patient\/([a-zA-Z0-9\-]+)\/?/.exec(location.pathname);
```

Comment:
`+` is "greedy" (a regex term), so `.+` will actually capture the trailing `/chart` as part of the capture group. My solution is to whitelist which characters are allowed within patient uuids (notably not including the `/` character)